foundry.toml

```diff
@@ -32,4 +32,4 @@ sepolia = "https://ci-sepolia-l1-archive.optimism.io" # Must have 'circleci_ip_r
 [profile.default.rpc_endpoints]
 localhost = "http://127.0.0.1:8545"
 mainnet = "https://ethereum.publicnode.com"
-sepolia = "https://ethereum-sepolia-rpc.publicnode.com"
+sepolia = "https://gateway.tenderly.co/public/sepolia"
```

Comment:
this is an archive node.

Reply:
`testRegressionCallDataMatches_OPCMUpgradeVxyz` in `Regression.t.sol` requires an archive node with the default profile.